packages/core/index.ts

```diff
@@ -209,6 +211,14 @@ export function createEnv<
       }
       return target[prop as keyof typeof target];
     },
+    set(_target, prop) {
+      // Readonly - this is the error message you get from assigning to a frozen object
+      throw new Error(
+        typeof prop === "string"
+          ? `Cannot assign to read only property ${prop} of object #<Object>`
+          : `Cannot assign to read only property of object #<Object>`
+      );
+    },
```

Comment:
what do you think about this?

Reply:
i dont like it for the reasons that I outlined in my previous post

Reply:
reverted change